### PR TITLE
Add a control test so we can properly compare how the new nav behaves

### PR DESCRIPTION
**Pull request by NataliaLKB**
## What does this change?
Had a chat with @JonNorman yesterday and turns out we do still need to have a control test when we are doing serverside tests 😞 .

This adds one for the new header test.

## What is the value of this and can you measure success?
We can measure things properly

## Does this affect other platforms - Amp, Apps, etc?
No

## Screenshots
n/a

## Tested in CODE?
n/a

### CODE REVIEW

File: common/app/mvt/MultiVariateTesting.scala [hunk 1]
```diff
@@ -77,7 +89,8 @@ object ActiveTests extends ServerSideABTests {
   val tests: Seq[TestDefinition] = List(
     CommercialClientLoggingVariant,
     CommercialGalleryBannerAds,
-    ABNewDesktopHeader
+    ABNewDesktopHeader,
```
Comment (gustavpursche) [inline common/app/mvt/MultiVariateTesting.scala]: Should this be called `ABNewDesktopHeaderVariant` instead?

Comment (NataliaLKB) [inline common/app/mvt/MultiVariateTesting.scala]: It might make it more clear. I will change it now.

**Review by gu-stav — APPROVED** [review]
Looks good to me now! I assume the Opt-In link still works?

**Comment by NataliaLKB** [comment]
> I assume the Opt-In link still works

Yep! The header name/value is unchanged.

**Pull request merged**